mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -780,6 +785,45 @@ class WidgetAdapter(
         }
     }
 
+    class ButtongridViewHolder internal constructor(private val initData: ViewHolderInitData) :
+        LabeledItemBaseViewHolder(initData, R.layout.widgetlist_buttongriditem), View.OnClickListener {
+        private val table: GridLayout = itemView.findViewById(R.id.widget_content)
+
+        override fun bind(widget: Widget) {
+            super.bind(widget)
+            val mappings = widget.mappings.filter { it.column != 0 && it.row != 0 }
+            table.rowCount = mappings.maxOfOrNull { it.row } ?: 0
+            table.columnCount = mappings.maxOfOrNull { it.column } ?: 0
+            (0..<table.rowCount).forEach { row ->
```

Comment:
What kind of syntax is this (`..<table`)?

Reply:
0..table.RowCount - 1

Reply:
So the same as `0 until table.rowCount`? If so, I'd consider that a little less obscure ;-)

Reply:
Yes, it's the same. I changed it.